Packs/GCP-Enrichment-Remediation/Playbooks/playbook-Cloud_Response_-_GCP.yml

```diff
@@ -0,0 +1,2049 @@
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "25":
+    id: "25"
+    taskid: 050459fa-9edc-4342-8f51-ec4b535e1090
+    type: condition
+    task:
+      id: 050459fa-9edc-4342-8f51-ec4b535e1090
+      version: -1
+      name: Should remediate automatically?
+      description: Checks if the user chose automatic remediation.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "34"
+      "yes":
+      - "26"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.autoResourceRemediation
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1980,
+          "y": 860
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "26":
+    id: "26"
+    taskid: dcd2adf0-93e1-4672-8cd6-29469a65105b
+    type: condition
+    task:
+      id: dcd2adf0-93e1-4672-8cd6-29469a65105b
+      version: -1
+      name: Continue by remediation type
+      description: Checks for the user's chosen remediation type.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      Delete:
+      - "27"
+      Stop:
+      - "28"
+    separatecontext: false
+    conditions:
+    - label: Delete
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.resourceRemediationType
+            iscontext: true
+          right:
+            value:
+              simple: Terminate
+          ignorecase: true
+    - label: Stop
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.resourceRemediationType
+            iscontext: true
+          right:
+            value:
+              simple: Stop
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1980,
+          "y": 1605
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "27":
+    id: "27"
+    taskid: b7c92358-4c34-42ab-844a-87cc9cbac517
+    type: regular
+    task:
+      id: b7c92358-4c34-42ab-844a-87cc9cbac517
+      version: -1
+      name: Delete the instance
+      description: Deletes the specified instance resource. For more information, see Stopping or Deleting an Instance.
```

Comment:
Where is Stopping and Deleting an Instance?  
From where can I get more information?

Reply:
As mentioned in Slack, those are OOTB command descriptions